pyproject.toml

```diff
@@ -0,0 +1,77 @@
+[build-system]
+requires = [
+    "poetry>=1.0.0",
+]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "sqlalchemy_exasol"
+version = "3.0.0"
+description = "EXASOL dialect for SQLAlchemy"
+readme = "README.rst"
+license = "License :: OSI Approved :: BSD License"
+authors = [
+    "Exasol AG <opensource@exasol.com>",
+    "Blue Yonder GmbH",
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Operating System :: POSIX :: Linux",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Education",
+    "Intended Audience :: Information Technology",
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: SQL",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Topic :: Database",
+    "Topic :: Software Development",
+    "Topic :: Software Development :: Libraries",
+    "Topic :: Software Development :: Libraries :: Application Frameworks",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Topic :: Scientific/Engineering :: Information Analysis",
+]
+keywords = [
+    "exasol",
+    "sql",
+    "sqlalchemy",
+    "data science",
+    "database",
+]
+repository = "https://github.com/exasol/sqlalchemy-exasol"
+homepage = "https://www.exasol.com/"
+
+include = [
+    "README.rst",
+    "CHANGES.md",
+    "LICENSE",
+]
+exclude = []
+
+
+[tool.poetry.dependencies]
+python = ">=3.8,<4.0"
+SQLAlchemy = ">=1.3.24,<1.4"
+pyodbc = ">=4.0.32"
+packaging = "^21.3"
+turbodbc = {path = "../../Downloads/turbodbc-4.5.3.tar.gz"}
```

Comment:
this needs to be changed later

Reply:
:white_check_mark:  agreed, currently locally needed because I wait on the turbodbc fix.